hammer/technology/sky130/qrc_lef.map

```diff
@@ -0,0 +1,14 @@
+extraction_setup \
+-technology_layer_map \
+met1 metal1 \
+met2 metal2 \
+met3 metal3 \
+met4 metal4 \
+met5 metal5 \
+li1 li \
```

Comment:
is there a reason li1 is after all the metals?

Reply:
no

Reply:
no, this was just the order I read it out of innovus or whatever when I created it. will put it at the top